src/cli-storage.ts

```diff
@@ -0,0 +1,19 @@
+import fs from "node:fs";
+import os from "node:os";
+
+const __confDir = os.homedir() + "/.baseplate";
+if (!fs.existsSync(__confDir)) {
```

Comment:
Do all executions of baseplate cli require file system storage? If not, it might be better to move this into the `storageProvider` function. Interacting with the file system makes things a bit harder to test and might be an unexpected side effect for users of baseplate cli

Reply:
All commands need it as now we support, stored token in addition to env var and argument